pylab/website/templates/website/about.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% load trans blocktrans from i18n %}
+
+
+{% block content %}
+  <div class="about">
+    <h1>{% trans "About Summer Python Workshop" %}</h1>
+
+    <p>{% blocktrans %}During this event everybody can learn how to program in
+    Python and Django by creating <a href="http://pylab.lt">pylab.lt</a>
+    website.{% endblocktrans %}</p>
```

Comment:
But we chose to work on pylab.lt till the end of the summer. Probably should add additional sentence to Python workshops policy about how events are organised, i.e. initial meeting, selection of project.